webpack.config.js

```diff
@@ -420,7 +420,7 @@ if (targetIsDevServer) {
             }],
             port: 9005,
             watchContentBase: true,
-            writeToDisk: false,
+            writeToDisk: true,
```

Comment:
Was this change intentional?

Reply:
Yes, in order to get the debugger working with VSCode, but I don't think I should slip this in a PR about a different feature, so I'll take it out.